images/installer-aws/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM openshift/origin-ansible
+
+USER root
+
+RUN yum install -y python2-boto python2-pip && \
+    pip install boto3
+
+RUN sed -i 's/inventory_ignore_extensions =\(.*\)/inventory_ignore_extensions =\1, .ini/g' /usr/share/ansible/openshift-ansible/ansible.cfg
```

Comment:
Note the above line is a "workaround" until we get the updated image from `openshift-ansible` with PR https://github.com/openshift/openshift-ansible/pull/6470